src/commands/deploy/deploy.ts

```diff
@@ -185,7 +185,7 @@ async function deployToSasViya(
           .map((i: { line: string }) => i.line)
           .join(os.EOL)
       : JSON.stringify(executionResult.log).replace(/\\n/g, os.EOL)
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
`e` shouldn't be of type `any`, because it has to be an object with property `message`

Reply:
We cannot define type of error in catch, it has to be `any` or `unknown`